extensions/bloodsugar-converter/src/convert-from-mg-dl-to-mmol-l.ts

```diff
@@ -0,0 +1,13 @@
+import { LaunchProps, showToast } from "@raycast/api";
+import { showFailureToast } from "@raycast/utils";
+
+export default function Command(props: LaunchProps<{ arguments: Arguments.ConvertFromMgDlToMmolL }>) {
+  if (!Number.isInteger(props.arguments.value)) {
```

Comment:
(non-blocking) This command will become way more extensible if in addition to supplying arguments, you can also use [Clipboard](https://developers.raycast.com/api-reference/clipboard#clipboard.readtext) and/or [getSelectedText](https://developers.raycast.com/api-reference/environment#getselectedtext). For example, I can select a number, press an assigned hotkey to the command and get the converted value. You can create new commands for these functionality extensions or you can add those helpers here itself. What do you think?